Movies-Selection/Movies-Selection/Helpers/Router/Router.swift

```diff
@@ -0,0 +1,39 @@
+// Router.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+/// Родительский протокол для Роутера
+protocol RouterProtocol {
+    var navigationController: UINavigationController? { get set }
+    var builder: BuilderProtocol? { get set }
+    func routerSetup(navigationController: UINavigationController, builder: BuilderProtocol)
+    func initialViewController()
+    func showDetailModule(movieID: Int?)
+}
+
+/// Router - Роутер для навигации модулей
+final class Router: RouterProtocol {
+    // MARK: - Internal Properties
+
+    var navigationController: UINavigationController?
+    var builder: BuilderProtocol?
+
+    // MARK: - Internal Methods
+
+    func routerSetup(navigationController: UINavigationController, builder: BuilderProtocol) {
```

Comment:
```suggestion
    func routerSetup(navigationController: UINavigationController, builder: BuilderProtocol) {
```
лучше инъекцию через конструктор сделать